### PR TITLE
doc: make set up step explicit

### PR DESCRIPTION
**Pull request by jmoreira-valory**
## Proposed changes

Make set up step explicit in the demo instructions so that it indicates the specific version of Open Autonomy that the service currently supports.

This is done in order to avoid situations where the Open Autonomy framework has been updated with a backwards-incompatible change (and thus the service cannot refer to the "normal" set up instructions).

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by Adamantios — CHANGES_REQUESTED** [review]
Please rebase on `chore/bump`.

**Review by Adamantios — APPROVED** [review]

**Review by dvilelaf — DISMISSED** [review]

**Comment by Adamantios** [comment]
Merging my PR and deleting my branch automatically closed this PR @jmoreira-valory . Could you please re-open it?

**Comment by Adamantios** [comment]
Are you sure you rebased? It seems there are things missing in your file's version compared to the version on `main`.

File: docs/index.md [hunk 1]
```diff
@@ -8,22 +8,32 @@ In the live demo, the predictions are currently sent via a POST message to an HT
...
+In order to run a local demo of the ML APY Prediction Oracle service:
 
-1. Fetch the ML APY Prediction Oracle service.
+1. [Set up your system](https://docs.autonolas.network/open-autonomy/guides/set_up/) to work with the Open Autonomy framework. We recommend that you use these commands:
+
+    ```bash
+    mkdir your_workspace && cd your_workspace
+    touch Pipfile && pipenv --python 3.10 && pipenv shell
+
+    pipenv install open-autonomy[all]==0.9.0
```
Comment (Adamantios) [inline docs/index.md:19]: I do not see a reason to use an empty `Pipfile` and then install the package like this. In general, why use `Pipenv` at all?

Comment (jmoreira-valory) [reply docs/index.md:19]: IMO I agree we should not instruct to use Pipenv, and not use any Pipenv (or similar) command in the tutorials.

We can *recommend* using Pipenv, and ultimately, it's user's decision if he wants to install the library in a virtual environment or not.

If we want to make this change, we need to make it in the Toolkits, as well as in the main Open Autonomy docs.

Comment (Adamantios) [reply docs/index.md:19]: Agreed! Sounds too involved, we need to change it in too many places, let's consider that in the future.

Comment (Adamantios) [reply docs/index.md:19]: Addressed on https://github.com/valory-xyz/apy-oracle/pull/45#discussion_r1115496043.

**Review by Adamantios — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (DavidMinarsch) [reply docs/index.md:19]: We use Pipenv as otherwise there’s no separation of environments

**Pull request merged**